exposee-app/server.js

```diff
@@ -65,6 +114,33 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
+app.post('/broadcast', async (req, res)=> {
+  try {
+    const {url, description, duration, userId, api_key} = req.body;
```

Comment:
it's generally bad practice to pass in the userId from the req.body, the userId should be coming from the server side, eg. session

It's a bad practice because now anybody can create a Video record with any userId if they know a bit of js and how to use postman